policyengine_us/parameters/gov/states/la/tax/income/exemptions/blind.yaml

```
@@ -0,0 +1,13 @@
+description: Louisiana provides an exemption of this amount for blind filers.
```

Do the same for others. Leave the labels with the `exemption` term to map to forms.
```suggestion
description: Louisiana reduces taxable income by this amount for each blind head or spouse.
```